src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapConnection.Linux.cs

```diff
@@ -127,6 +127,8 @@ private int BindSasl()
             Marshal.StructureToPtr(defaults, ptrToDefaults, false);
             try
             {
+                // Bump up the protocol version because ldap_sasl_interactive_bind requires LDAP V3 else it returns LDAP_NOT_SUPPORTED and this ends up throwing LdapException: The feature is not supported.
+                SessionOptions.ProtocolVersion = 3;
```

Comment:
For non-OSX, this must be working before the change above -- will this break V1\V2 users in those cases?

Reply:
Actually I don't have a Linux machine to test. I tried to setup Kerberos + OpenLDAP in Docker to test on Linux but I couldn't create a working environment.